src/router/Router.tsx

```diff
@@ -169,41 +171,51 @@ export const RoutesProvider = () => {
   if (isSaas) {
     routes.push(
       {
-        element: <LayoutAuth />,
+        path: 'account',
+        element: (
+          <SuspenseLoader>
+            <SaasAuthProtectedRoute />
+          </SuspenseLoader>
+        ),
         children: [
           {
+            element: <LayoutAuth />,
             children: [
               {
-                path: 'signin',
-                element: (
-                  <SuspenseLoader>
-                    <SignIn />
-                  </SuspenseLoader>
-                ),
-              },
-              {
-                path: 'signup',
-                element: (
-                  <SuspenseLoader>
-                    <SignUp />
-                  </SuspenseLoader>
-                ),
-              },
-              {
-                path: 'account/recovery',
-                element: (
-                  <SuspenseLoader>
-                    <ForgotPassword />
-                  </SuspenseLoader>
-                ),
-              },
-              {
-                path: 'account/verify',
-                element: (
-                  <SuspenseLoader>
-                    <Verify />
-                  </SuspenseLoader>
-                ),
+                children: [
+                  {
+                    path: 'signin',
```

Comment:
@selankon are you sure that signin and signup should under the `SaasAuthProtectedRoute`?  
Also maybe they signup part should be under `SaaSProtectedRoute`, meaning to have an address?
Btw the nameing begins to become quite vague

Reply:
`SaasAuthProtectedRoute` in reality is a middleware that protect that if  you already logged in you cannot `signup`, `login`, `recoverMail`... withou a previous logout to avoid strange bugs. 

https://github.com/vocdoni/ui-scaffold/blob/026f62cfa70f50d7a8b82ed1674a38abeefe52c7/src/router/SaasAuthProtectedRoute.tsx#L12-L14

Agree that `SaasAuthProtectedRoute` is not the best name